libmscore/rendermidi.cpp

```diff
@@ -731,6 +735,478 @@ bool Score::isSubdivided(ChordRest* chord, int swingUnit)
             return false;
       }
 
+void renderTremolo(Chord *chord, QList<NoteEventList> & ell){
```

Comment:
Hi Nicolas,  as far as the indentation.  I can fix it, in as much as I understand the indentation rules.
However, I tried to preserve the indentation which was already their.  I basically just move some code in an overbloated function
into several smaller functions, making the original function much easier to read. 


> On 18 May 2015, at 12:36, Nicolas Froment <notifications@github.com> wrote:
> 
> In libmscore/rendermidi.cpp <https://github.com/musescore/MuseScore/pull/1946#discussion_r30496906>:
> 
> > @@ -731,6 +735,478 @@ bool Score::isSubdivided(ChordRest* chord, int swingUnit)
> >              return false;
> >        }
> >  
> > +void renderTremolo(Chord *chord, QList<NoteEventList> & ell){
> Indentation of these new functions is completely off. There is no comment before the function. Idem for renderArpeggio()
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/musescore/MuseScore/pull/1946/files#r30496906>.
>